code/style.css

```diff
@@ -5,3 +5,326 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+body{
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color:rgb(136, 177, 177);
+}
+header{
+  width: 100%;
+}
+img{
+  object-fit: cover;
```

Comment:
wow, this is a great code to make it maintain its own ratio! Thanks for the great tip!